lib/components/map/endpoints-overlay.js

```diff
@@ -27,9 +36,11 @@ const mapStateToProps = (state, ownProps) => {
   // current query is no search is available.
   const activeSearch = getActiveSearch(state.otp)
   const query = activeSearch ? activeSearch.query : state.otp.currentQuery
+  const showUserSettings = state.otp.config.persistence && state.otp.config.persistence.enabled
```

Comment:
This same check is showing up in a bunch of places. It'd probably be a good idea to make a util function or selector for it.